src/components/AvailableImagesTile.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {
+  Card,
+  CardTitle,
+  CardBody,
+  Button,
+  Spinner,
+} from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+
+const AvailableImageTileBase = ({ children }) => {
+  return (
+    <Card>
+      <CardTitle>Available images</CardTitle>
+      {children}
+    </Card>
+  );
+};
```

Comment:
Extra return statement
```suggestion
const AvailableImageTileBase = ({ children }) => (
  <Card>
    <CardTitle>Available images</CardTitle>
    {children}
  </Card>
);
```